test-fixtures/spago-subpackage-test-cwd.purs

```diff
@@ -0,0 +1,17 @@
+module Subpackage.Test.Main where
+
+import Prelude
+
+import Effect (Effect)
+import Effect.Aff (launchAff_)
+import Effect.Class.Console (log)
+import Node.Encoding (Encoding(..))
+import Node.FS.Aff (readTextFile)
+
+main :: Effect Unit
+main = launchAff_ do
+  readTextFile UTF8 filePath >>= log
+
+-- DO NOT EDIT. This value is replaced in the corresponding spago test
+filePath :: String
+filePath = "FILE_PATH"
```

Comment:
```suggestion
filePath = "./foo.txt"
```

I think we don't need to do the whole replacing thing, but can just use a relative path?

Reply:
Looks like spago has historically had windows CI, so should this be (“.” <> Path.sep <> “file.txt”) instead?

Reply:
It might also work to just use `foo.txt`